src/hats/io/file_io/file_io.py

```diff
@@ -148,6 +149,15 @@ def write_dataframe_to_parquet(dataframe: pd.DataFrame, file_pointer):
     dataframe.to_parquet(file_pointer.path, filesystem=file_pointer.fs)
 
 
+def _parquet_bytes_io(file_pointer):  # pragma: no cover
+    if not isinstance(file_pointer, upath.implementations.http.HTTPPath):
+        return False
+    cache_options = file_pointer.fs.cache_options or {}
+    if "parquet_bytes_io" not in cache_options:
+        return False
+    return cache_options["parquet_bytes_io"]
```

Comment:
Given the effect of this option, I think that something like `parquet_precache_all_bytes` might be clearer than `parquet_bytes_io`, unless there is an existing naming convention to conform to.